lms/templates/student_account/login_and_register.html

```diff
@@ -22,6 +23,9 @@
             newrelic.addPageAction('xfinished');
         }
     </%static:require_module>
+    % if configuration_helpers.get_value('DISPLAY_TOS_IN_MODAL_ON_REGISTRATION_PAGE', False):
+    <script type="text/javascript" src="${static.url('js/student_account/tos_modal.js')}"></script>
+    % endif
```

Comment:
Just realized we will also need to add the changes to this template to the "themed" versions of this template in [edx/edx-themes](https://github.com/edx/edx-themes):

https://github.com/edx/edx-themes/blob/master/edx-platform/globalacademy.hms.harvard.edu/lms/templates/student_account/login_and_register.html

https://github.com/edx/edx-themes/blob/master/edx-platform/harvardxplus.harvard.edu/lms/templates/student_account/login_and_register.html

https://github.com/edx/edx-themes/blob/master/edx-platform/juilliardopenclassroom.com/lms/templates/student_account/login_and_register.html

https://github.com/edx/edx-themes/blob/master/edx-platform/mitprofessionalx.mit.edu/lms/templates/student_account/login_and_register.html

https://github.com/edx/edx-themes/blob/master/edx-platform/professionaleducation.wharton.upenn.edu/lms/templates/student_account/login_and_register.html